src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po

```diff
@@ -0,0 +1,291 @@
+# Bengali translations for humanize package.
+# Copyright (C) 2021 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the humanize package.
+# Wasi Master <arianmollik323@gmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-07-29 21:59+0600\n"
+"PO-Revision-Date: 2021-07-29 22:07+0600\n"
+"Last-Translator: U-WASI-PC\\Wasi Master <arianmollik323@gmail.com>\n"
+"Language-Team: Bengali\n"
+"Language: bn_BD\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```

Comment:
And we'll need the correct values in here.

Reply:
Please replace `INTEGER` and `EXPRESSION`

See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html which has examples for other language families.